### PR TITLE
- added function to show and hide barcode numbers

### PR DESCRIPTION
**Pull request by danieljausovec**
Please add this feature in the 7.0 version.

### CODE REVIEW

File: mpdf.php [hunk 1]
```diff
@@ -1416,229 +1418,229 @@ function _getPageFormat($format)
...
+				break;
+			} //	'Royal' format paperback size 153x234mm
 			default: {
-					$format = array(595.28, 841.89);
-					break;
-				}
+				$format = array(595.28, 841.89);
+				break;
+			}
 		}
```
Comment (finwe) [inline mpdf.php]: These changes have no connection to the theme of the PR, please keep as it were.

**Comment by finwe** [comment]
Thanks for the PR. However.

The PR is a mess. Please:
* If requesting a feature for 7.0, start it from the `development branch`
* Only change lines of code that have something to do with the scope of the PR, otherwise it is almost impossible to code-review the patch.
* It'd be great if the patch were accompanied with a simple unit test covering the affected lines

When ready, just forcepush your branch, the PR will update itself.

See also [the contributing guidelines][1].

[1]:https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md

File: mpdf.php (hunk 1 above)
Comment (danieljausovec) [inline mpdf.php]: I have nothing delete. Just added one IF.

`if ($this->showBarcodeNumbers)`

Comment (finwe) [inline mpdf.php]: Then your editor must have reformatted the rest of the code automagically. Please read the comment below the original pull request.

**Comment by danieljausovec** [comment]
I edited existing code. Can this be combined with the master?

**Comment by finwe** [comment]
No. Development branch is completely rewritten and no new features will be added to soon-to-be-legacy master/6.x branches.

Only serious fixes will be backported to older branches.

**Comment by danieljausovec** [comment]
Ok. When release new version 7.0? Is possible included this feature?

**Comment by finwe** [comment]
Yes, I will be glad to incorporate this feature to 7.0 (when it meets the standards hinted above). 

The 7.0 does not have a release date yet, progress can be in the [milestone 7.0](https://github.com/mpdf/mpdf/milestones/7.0).

**Pull request closed without merging**